youtube_dl/extractor/indavideo.py

```diff
@@ -9,8 +11,14 @@
 )
 
 
-class IndavideoEmbedIE(InfoExtractor):
```

Comment:
I understand, but what's the point if I want to delete one of the functions because its function merged to the other one. In my opinion this is **not changing of the name**, because before the commit there was 2 extractors: IndavideoIE and IndavideoEmbedIE, with the commit I've deleted the IndavideoEmbedIE